ci/nightly-build.sh

```diff
@@ -26,7 +26,8 @@ MVN="mvn -Dmaven.wagon.http.retryHandler.count=3 -B"
 CUDA_VER=${CUDA_VER:-cuda`nvcc --version | sed -n 's/^.*release \([0-9]\+\)\..*$/\1/p'`}
 PARALLEL_LEVEL=${PARALLEL_LEVEL:-4}
 USE_GDS=${USE_GDS:-ON}
-${MVN} clean package ${MVN_MIRROR}  \
+SANITIZER_CMD="compute-sanitizer --tool memcheck --target-processes all $SANITIZER_OPTS"
```

Comment:
Q: 
So should all cuda runtimes have included the compute-sanitizer tooling? do we need to do some prerequisites to make sure the ENV has required pkgs for test run?

Reply:
I think yes.
Checked cuda 11.5 and 11.8, both have this tool.